public/scripts/developerTools.js

```diff
@@ -0,0 +1,36 @@
+function ready(fn) {
+  if (document.readyState != "loading") {
+    fn();
+  } else {
+    document.addEventListener("DOMContentLoaded", fn);
+  }
+}
+ready(main);
+
+function main() {
+  function handleIntersection(entries, observer) {
+    entries.forEach((entry) => {
+      if (entry.isIntersecting) {
+        entry.target.classList.add("animate");
+      }
+    });
+  }
+
+  const observer = new IntersectionObserver(handleIntersection);
+
+  /* make the animation in mobile mode look smoother */
```

Comment:
Why does this make the animation smoother? What problem did it solve?

Reply:
@pkong-ds updated comment